src/cryptography/x509/verification.py

```diff
@@ -2,8 +2,56 @@
 # 2.0, and the BSD License. See the LICENSE file in the root of this repository
 # for complete details.
 
+from __future__ import annotations
+
+import datetime
+import typing
+
 from cryptography.hazmat.bindings._rust import x509 as rust_x509
+from cryptography.x509.general_name import DNSName, IPAddress
 
-__all__ = ["Store"]
+__all__ = ["Store", "Subject", "PolicyBuilder"]
 
 Store = rust_x509.Store
+
+Subject = typing.Union[DNSName, IPAddress]
+
+
+class PolicyBuilder:
+    def __init__(
+        self,
+        *,
+        subject: Subject | None = None,
+        time: datetime.datetime | None = None,
+    ):
+        self._subject = subject
+        self._time = time
+
+    @classmethod
+    def webpki(cls) -> PolicyBuilder:
+        return PolicyBuilder(time=datetime.datetime.now())
+
+    def subject(self, new_subject: Subject) -> PolicyBuilder:
+        """
+        Sets the expected certificate subject.
+        """
+        return PolicyBuilder(
```

Comment:
Our other builders raise exceptions if you try to set the same field 2x.

Reply:
https://github.com/pyca/cryptography/pull/9601/commits/9aec060009b03117bafa21b6d94bceed0beffedf